packages/web/src/components/nav/desktop/LeftNav.tsx

```diff
@@ -124,76 +125,82 @@ const LeftNav = (props: NavColumnProps) => {
     <nav id='leftNav' className={styles.leftNav}>
       {isElectron ? <RouteNav /> : null}
       <NavHeader />
-      <div
-        ref={navBodyContainerMeasureRef}
-        className={cn(styles.leftNavContent, {
-          [styles.show]: navLoaded,
-          [styles.dragScrollingUp]: dragScrollingDirection === 'up',
-          [styles.dragScrollingDown]: dragScrollingDirection === 'down'
-        })}
-      >
-        <Scrollbar
-          containerRef={(el: HTMLElement) => {
-            scrollbarRef.current = el
-          }}
-          className={styles.scrollable}
+      <ClientOnly>
```

Comment:
really cool how effectively this works

Reply:
Yeah it's simple too! Vike also provides a `ClientOnly` component that supports lazy loading and is more complex:
https://vike.dev/ClientOnly 
Didn't have a use for that yet though